### PR TITLE
Release 3.4.0

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: repos/katello.repo [hunk 1]
```diff
@@ -5,7 +5,7 @@ name=Katello @REPO_NAME@
 baseurl=https://fedorapeople.org/groups/katello/releases/yum/@REPO_VERSION@/katello/@DIST@/$basearch/
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-katello
```
Comment (jlsherrill) [inline repos/katello.repo:6]: th ename of the gpg key would need to be updated in this file

Comment (ehelms) [reply repos/katello.repo:6]: Why? I didn't change the name of the key when it gets deployed.

Comment (jlsherrill) [reply repos/katello.repo:6]: oh right, cause the source2 just gets copied.

**Comment by ehelms** [comment]
[test]

**Comment by jlsherrill** [comment]
[test]

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**